src/app/app-routing.module.ts

```diff
@@ -53,7 +57,11 @@ const appRoutes: Routes = [
         path: 'admin', canActivate: [AuthGuard, AdminGuard], children: [
           {path: '', component: AdminComponent, pathMatch: 'full'},
           {path: 'edit', component: AdminIssuerEditComponent},
```

Comment:
We should change routing for show/edit issuer because it doesn't make sense to have:
* show issuer: `/admin`
* edit issuer: `/admin/edit`

We are not showing and editing admin, but the issuer.
It would be better to have:
* `/admin` (pathMatch: 'full') -> redirectTo: 'issuer'
* `/admin/issuer` -> AdminIssuerComponent
* `/admin/issuer/edit` -> AdminIssuerEditComponent

After making modifications, check navigate/routerLink methods to have correct properties.